src/content/tutorials_src/005_savitzky_golay_temperature.fsx

```diff
@@ -0,0 +1,227 @@
+
+
+# Smoothing data with the Savitzky-Golay filter
+
+_Summary:_ This tutorial demonstrates how to access a public dataset for temperature data with [FSharp.Data](https://fsprojects.github.io/FSharp.Data/), how to smoothe the data points with 
+the Savitzky-Golay filter from [FSharp.Stats](https://fslab.org/FSharp.Stats/) and finally how to visualize the results with [Plotly.NET](https://plotly.net).
+
+## Introduction: 
+
+The Savitzky-Golay is a type of low-pass filter, particularly suited for smoothing noisy data. The main idea behind this approach is to make for each point a 
+least-square fit with a polynomial of high order over a odd-sized window centered at the point. One advantage of the Savitzky-Golay filter is that, portions 
+of high frequencies are not simply cut off, but are preserved due to the polynomial regression. This allows the filter to preserve properties of the distribution 
+such as relative maxima, minima and dispersion, which are usually distorted by flattening or shifting by conventional methods such as moving average.
+
+This is useful when trying to identify general trends in highly fluctuating data sets, or to smooth out noise to improve the ability to find minima and maxima of the data trend.
+To showcase this we will plot a temperature dataset from the ["Deutsche Wetterdienst"](https://www.dwd.de/DE/leistungen/klimadatendeutschland/klimadatendeutschland.html), 
+a german organization for climate data. We will do this for both the original data points and a smoothed version.
+
+<center>
+
+![windowed polynomial regression](https://upload.wikimedia.org/wikipedia/commons/8/89/Lissage_sg3_anim.gif)
+
+The image shows the moving window for polynomial regression used in the Savitzky-Golay filter [@wikipedia](https://upload.wikimedia.org/wikipedia/commons/8/89/Lissage_sg3_anim.gif)
+
+</center>
+
+## Referencing packages
+
+```fsharp
+// Packages hosted by the Fslab community
+#r "nuget: FSharp.Stats"
+// third party .net packages 
+#r "nuget: FSharp.Data"
+#r "nuget: Plotly.NET, 2.0.0-preview.6"
+#r "nuget: Plotly.NET.Interactive, 2.0.0-preview.6"
+```
+
+*)
+
+
+(**
+## Loading data
+
+We will start by retrieving the data. This is done with the [FSharp.Data](https://fsprojects.github.io/FSharp.Data/) package 
+and will return a single string in the original format.
+*)
+
+// Get data from Deutscher Wetterdienst
+// Explanation for Abbreviations: https://www.dwd.de/DE/leistungen/klimadatendeutschland/beschreibung_tagesmonatswerte.html
+let rawData = FSharp.Data.Http.RequestString @"https://www.dwd.de/DE/leistungen/_config/leistungsteckbriefPublication.txt?view=nasPublication&nn=16102&imageFilePath=8524108716561010473591414984380285860348440627905136707616031491601294722759953476578874542442119146105183517945099480938740343906988335742522410754862714750285564712567437754031667430876229001517155807795323254373913159234471816274144126039683579197&download=true"
```

Comment:
If the data is publicly available, please submit it as a PR to the [dataset repo](https://github.com/fslaborg/datasets) with correct attribution. If not, please comment here, the rest LGTM

Reply:
Done!